MIGRATING.md

```diff
@@ -0,0 +1,92 @@
+# Migrating from v1 to v2
+
+## Client Initialization
+
+```typescript
+// In both v1 and v2, it is not required to pass an api key. It can be retrieved from the environment variable LMNT_API_KEY.
```

Comment:
The API key can be retrieved using the LMNT_API_KEY environment variable or instead passed as an argument.